src-java/northbound-service/northbound/src/test/java/org/openkilda/northbound/converter/FlowMapperTest.java

```diff
@@ -110,7 +114,8 @@ public void testFlowRequestV2Mapping() {
         assertEquals(FlowEncapsulationType.TRANSIT_VLAN, flowRequest.getEncapsulationType());
         assertEquals(DESCRIPTION, flowRequest.getDescription());
         assertEquals(BANDWIDTH, flowRequest.getBandwidth());
-        assertEquals(LATENCY * 1000000L, (long) flowRequest.getMaxLatency()); // ms to ns
+        assertEquals(LATENCY * MS_TO_NS_MULTIPLIER, (long) flowRequest.getMaxLatency()); // ms to ns
```

Comment:
[TimeUnit](https://docs.oracle.com/en/java/javase/12/docs/api/java.base/java/util/concurrent/TimeUnit.html) can be used for these kinds of conversions.